### PR TITLE
adding tax household group persistance models

### PR DESCRIPTION
**Pull request by nisanthyaganti9**

### CODE REVIEW

File: app/domain/operations/premium_credits/find_csr_value.rb [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      def handle_native_american_csr
+        return unless FinancialAssistanceRegistry.feature_enabled?(:native_american_csr)
+
+        @subjects.each do |subject|
+          @csr_hash[subject.gid.split('/').last] = 'limited' if subject.person.indian_tribe_member
+        end
+
+        family_members_with_ai_an = @subjects.map(&:person).select(&:indian_tribe_member).map(&:id).map(&:to_s)
+
+        @subjects = @subjects.reject { |subject| family_members_with_ai_an.include? subject.person_id.to_s }
```
Comment (saikumar9) [inline app/domain/operations/premium_credits/find_csr_value.rb:59]: These lines are not needed as you are already setting the CSR Hash above.

File: app/models/benefit_coverage_period.rb [hunk 2]
```diff
@@ -149,26 +149,43 @@ def earliest_effective_date
   #
   # @param hbx_enrollment_members [ Array ] the list of enrolling members
   # @param coverage_kind [ String ] the benefit type.  Only 'health' is currently supported
-  # @param tax_household [ TaxHousehold ] the tax household members belong to if eligible for financial assistance
+  # @param group_member_premium_credit [ GroupMemberPremiumCredit ] if eligible for financial assistance
```
Comment (saikumar9) [inline app/models/benefit_coverage_period.rb:152]: Update Comment appropriately.

**Review by saikumar9 — CHANGES_REQUESTED** [review]
There are just a couple of changes. These are not blockers.

**Review by saikumar9 — DISMISSED** [review]
Changes look good!

**Review by saikumar9 — DISMISSED** [review]
LGTM!

**Review by harshaellanki — APPROVED** [review]

**Pull request merged**